test/e2e/ctrcfg_test.go

```diff
@@ -104,7 +106,7 @@ func runTestWithCtrcfg(t *testing.T, testName, regexKey, expectedConfValue strin
 	// ensure config rolls back as expected
 	waitForConfigAndPoolComplete(t, cs, poolName, oldMCConfig.Name)
 
-	restoredConfValue := getValueFromCrioConfig(t, cs, node, regexKey)
+	restoredConfValue := getValueFromCrioConfig(t, cs, node, regexKey, false)
```

Comment:
AFAICS this no longer tests that rollback actually rolls back — the 00-default configuration is never changed. This should be testing that the override file is not present, or does not contain that key.

Reply:
Fixed